### PR TITLE
quincy: doc/mgr: add prompt directives to dashboard.rst

### PR DESCRIPTION
**Pull request by zdover23**
This commit adds prompt directives (.. prompt:: bash $) to
the commands in dashboard.rst.

There are several ".. include::" directives in the dashboard.rst
file, which means that part of this page is sourced from elsewhere
than the dashboard.rst file. Because I have not yet added prompt
directives to those files, there is an inconsistency in the rendering
of this file. Most of the commands on this page have unselectable
prompts (unselectable prompts are the prompts that don't get added to
the buffer when you copy them to one of the clipboards). But the
commands on this page that come from those ".. include::" directives
do not yet have unselectable prompts.

This file is over 1600 lines long. It was perhaps not optimally wise
of me to have edited all of it in one fell swoop. It took many hours,
and carefully checking it will probably take at least one hour. I
suggest that whoever reviews this should not spend much time on it,
but should instead make a quick pass over the page and make sure that
it looks passable.

The English syntax on this page (and throughout the Dashboard doc-
umentation) will be tightened to remove ambiguity and to improve
readability in the near future, so hold all English-language-related
comments for a future pull request.

Signed-off-by: Zac Dover <zac.dover@gmail.com>
(cherry picked from commit d8064b46815e5e5c3c173f81d72b21c0a8949db7)

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
The sibling of this PR, PR#47823, has dependencies that are, as of the writing of this note, unmet. The structure of dashboard.rst changed between Pacific and Quincy, and tedious, careful human judgement will be necessary.

**Comment by zdover23** [comment]
> The sibling of this PR, PR#47823, has dependencies that are, as of the writing of this note, unmet. The structure of dashboard.rst changed between Pacific and Quincy, and tedious, careful human judgement will be necessary.

The RST in the includes has been updated to include unselectable prompts in the following PR: https://github.com/ceph/ceph/pull/47843.

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -480,9 +518,11 @@ The following process describes how to configure Grafana and Prometheus
 manually. After you have installed Prometheus, Grafana, and the Node exporter
 on appropriate hosts, proceed with the following steps.
 
-#.  Enable the Ceph Exporter which comes as Ceph Manager module by running::
+#.  Enable the Ceph Exporter which comes as Ceph Manager module by running:
```
Comment (anthonyeleven) [inline doc/mgr/dashboard.rst:521]: ,which

module,

File: doc/mgr/dashboard.rst [hunk 2]
```diff
@@ -596,18 +644,22 @@ The format of the URL : `<protocol>://<IP-address>:<port>`
 If you are using a self-signed certificate for Grafana,
 disable certificate verification in the dashboard to avoid refused connections,
 which can be a result of certificates signed by an unknown CA or that do not
-match the host name::
+match the host name:
```
Comment (anthonyeleven) [inline doc/mgr/dashboard.rst:647]: I’m a “hostname” guy.  Ymmv.

File: doc/mgr/dashboard.rst [hunk 3]
```diff
@@ -659,9 +713,11 @@ process can be performed by an existing Identity Provider (IdP).
   Please ensure that this library is installed on your system, either by using
   your distribution's package management or via Python's `pip` installer.
 
-To configure SSO on Ceph Dashboard, you should use the following command::
+To configure SSO on Ceph Dashboard, you should use the following command:
```
Comment (anthonyeleven) [inline doc/mgr/dashboard.rst:716]: on the?

File: doc/mgr/dashboard.rst [hunk 4]
```diff
@@ -1088,21 +1211,27 @@ If the dashboard is behind a load-balancing proxy like `HAProxy <https://www.hap
...
-instead of redirecting to the active dashboard::
+instead of redirecting to the active dashboard:
+
+.. prompt:: bash $
 
-  $ ceph config set mgr mgr/dashboard/standby_behaviour "error"
+   ceph config set mgr mgr/dashboard/standby_behaviour "error"
 
-To reset the setting to default redirection, use the following command::
+To reset the setting to default redirection, use the following command:
```
Comment (anthonyeleven) [inline doc/mgr/dashboard.rst:1220]: default (redirection)

?

File: doc/mgr/dashboard.rst [hunk 5]
```diff
@@ -1088,21 +1211,27 @@ If the dashboard is behind a load-balancing proxy like `HAProxy <https://www.hap
...
+.. prompt:: bash $
+
+   ceph config set mgr mgr/dashboard/standby_behaviour "redirect"
 
 Configure the error status code
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 When redirection is disabled, you may want to customize the HTTP status
-code of standby dashboards. To do so you need to run the command::
+code of standby dashboards. To do so you need to run the command:
```
Comment (anthonyeleven) [inline doc/mgr/dashboard.rst:1230]: s/you need to//

File: doc/mgr/dashboard.rst [hunk 6]
```diff
@@ -1221,9 +1359,11 @@ Accessing the Dashboard
 If you are unable to access the Ceph Dashboard, run the following
 commands:
 
-#. Verify the Ceph Dashboard module is enabled::
+#. Verify the Ceph Dashboard module is enabled:
```
Comment (anthonyeleven) [inline doc/mgr/dashboard.rst:1362]: that the

**Review by anthonyeleven — APPROVED** [review]
See suggestions

**Review by nizamial09 — APPROVED** [review]

**Comment by nizamial09** [comment]
aah i was reviewing this yesterday and forgot to approve. I didn't see its merged since my page wasn't refreshed :laughing:

**Pull request merged**